docusaurus/docs/create-a-plugin/extend-a-plugin/fetch-data-from-frontend.md

```diff
@@ -219,34 +269,23 @@ logging = true
 
 With this configuration, the Grafana server output shows the requests going out to your API from the data proxy.
 
-### Send special headers using the data proxy
+## Send special headers using the data proxy
```

Comment:
I think this is covered here as well so maybe not needed? https://grafana.com/developers/plugin-tools/create-a-plugin/extend-a-plugin/add-authentication-for-data-source-plugins#add-http-headers-to-a-proxy-route

Reply:
It is indeed but I decided to duplicate the information to have it at hand in a single place ,Maybe a better idea to create a link to that page?

Reply:
Yes, link is a good idea